app/src/main/res/drawable/thumb_terraform_cloud.xml

```diff
@@ -0,0 +1,32 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="256dp"
+    android:height="256dp"
```

Comment:
As per Android Studio considerations for SVGs, it is recommended to limit a vector image to a maximum of 200 x 200 dp; otherwise, it can take too long to draw. See: https://developer.android.com/studio/write/vector-asset-studio#when

Reply:
You can do this scaling, e.g. with Inkscape.

Reply:
I scaled it, let me know if it's ok now! Thanks for the link to the docs.